packages/shared/src/contracts/registry.ts

```diff
@@ -4,33 +4,38 @@ import { ChainId } from '../web3';
 type Contracts = {
   readonly SUBSCRIPTION_MANAGER: ChecksumAddress | undefined;
   readonly COORDINATOR: ChecksumAddress | undefined;
+  readonly GLOBAL_ALLOW_LIST: ChecksumAddress | undefined;
 };
 
 const POLYGON: Contracts = {
   SUBSCRIPTION_MANAGER: '0xB0194073421192F6Cf38d72c791Be8729721A0b3',
   COORDINATOR: undefined,
+  GLOBAL_ALLOW_LIST: undefined,
 };
 
 const MUMBAI: Contracts = {
   SUBSCRIPTION_MANAGER: '0xb9015d7b35ce7c81dde38ef7136baa3b1044f313',
   COORDINATOR: '0x8E49989F9D3aD89c8ab0de21FbA2E00C67ca872F',
+  GLOBAL_ALLOW_LIST: '0x7b521E78CFaf55fa01433181d1D636E7e4b73243',
```

Comment:
We should be careful here. Both `tapir` and `lynx` use Mumbai but have different GlobalAllowList contracts:
- Lynx: `0x7b521E78CFaf55fa01433181d1D636E7e4b73243`
- Tapir: `0x5Ff235481bbDBF5dFCC908d987506FA6a8696678`

How would this resolution work currently? Keying by "domain" is better than solely by blockchain/chain_id.

Reply:
Will address post https://github.com/nucypher/nucypher-ts/pull/314 changes